### DISCUSSION TITLE
Error while simulating. stderr: The process cannot access the file because it is being used by another process.

### DISCUSSION THREAD

**Discussion by ValeriolfBH** [urbanopt/urbanopt-cli#464] [Q&A]
Hi! When I try to run a simulation using UrbanOPT within Grasshopper I get this error message:

Error running command: 'C:/URBANopt-cli-0.10.0/OpenStudio/bin/openstudio.exe --bundle 'C:/Users/vlanfranco/Desktop/Dragonfly/Gemfile' --bundle_path 'C:/Users/vlanfranco/Desktop/Dragonfly/.bundle/install/' run -w 'C:/Users/vlanfranco/Desktop/Dragonfly/run/honeybee_scenario/Building_8_bda48746/in.osw' 2>&1 > "C:/Users/vlanfranco/Desktop/Dragonfly/run/honeybee_scenario/Building_8_bda48746/in.osw.log"'
stdout:
stderr: The process cannot access the file because it is being used by another process.

Any solutions to this?

**Comment by kflemin** [comment]
Hello,
For running UO via Dragonfly, your best bet is to check the Ladybug Tools forum: https://discourse.ladybug.tools/
